libraries/common/include/common/constants.hpp

```diff
@@ -26,6 +26,8 @@ const uint32_t kPeriodicEventsThreadCount{2};
 
 const uint64_t kMinTxGas{21000};
 
+constexpr uint32_t kMinTransactionPoolSize{30000};
```

Comment:
For any kind of high load tps testing this should be much bigger. If we are going for 10k tps, my recommendation would be that max size of the pool should be at least 200k. @justinsnapp what do you think?

Reply:
Also maybe we should have two constants, one for default value and one for min value, min can stay 30k but for default I would put 200k

Reply:
That's the idea, right now 30k is min but we can specify any above that